message_app/__init__.py

```diff
@@ -36,43 +36,44 @@ def create_app(test_config=None, name=__name__):
         # Some data for real-time testing purpose and will be DELETED before deployment.
         # Delete the current database or comment out this part if there is any conflict
         # Uncomment this part for real-time testing
-        # try:
-        #     # User
-        #     password_hash_1 = hash_pw("1")
-        #     password_hash_2 = hash_pw("2")
-        #     password_hash_3 = hash_pw("3")
-        #     user1 = User(username="user1", email="email11@test.com", password_hash=password_hash_1)
-        #     user2 = User(username="user2", email="email22@test.com", password_hash=password_hash_2)
-        #     user3 = User(username="user3", email="email33@test.com", password_hash=password_hash_3)
-        #     User.insert(user1)
-        #     User.insert(user2)
-        #     User.insert(user3)
+        if not test_config:
```

Comment:
@vuongtranluc If we run the app under non-test mode, we will have these users to use upon app initialization. It is convenient to have, but you can do whatever you want here. This might be removed in the future.
@khuolo01 nothing to change. just some explanation.